ViewControllers/MainTabBarController.swift

```diff
@@ -14,6 +14,8 @@ class MainTabBarController: UITabBarController {
     // MARK: ViewDidLoad
     override func viewDidLoad() {
         super.viewDidLoad()
+        DashXUtils.trackEventClient()
+        DashXUtils.client1.track("TestEventFromiOSApp")
```

Comment:
Let's not have events in the code which aren't also present in other demo apps...

Reply:
Sure, will remove these track events